apps/aggiemap-angular-e2e/src/support/index.d.ts

```diff
@@ -21,5 +21,9 @@ declare namespace Cypress {
      checkLink(elementText: string, link: string): Chainable<Element>
      containsAnyText(element: string): Chainable<Element>
      checkMenuItem(num: string, name:string): Chainable<Element>
+     checkPopUp(): Chainable<Element>
+     checkNavPanel(): Chainable<Element>
+     checkMapIcon(): Chainable<Element>
+     mockGeolocation(): Chainable<Element>
```

Comment:
All of these command typings shouldn't be in this file. Move back to commands. 

Also, document your functions and make sure the type returns are accurate.